src/components/InputTodo.jsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/prop-types */
+import { useState } from "react";
+
+const InputTodo = ({ addTodoItem }) => {
+  const [title, setTitle] = useState("");
+  const [msg, setMsg] = useState("");
+  const handelChange = (e) => {
+    setTitle(e.target.value);
+  };
+
+  const handelSubmit = (e) => {
+    e.preventDefault();
+    // console.log(title);
+    // console.log(title.trim());
+    if (title.trim()) {
```

Comment:
It will work without `trim()`
`trim()` only affects on console, not DOM

Reply:
Thanks for this information 💯 👍